### PR TITLE
Mask guidance updated

### PR DESCRIPTION
**Pull request by brunobowden**
- Advice sourced from this page:
- https://www.who.int/emergencies/diseases/novel-coronavirus-2019/question-and-answers-hub/q-a-detail/q-a-on-covid-19-and-masks
- Partial fix for #1511

## Screenshots

<details>
<summary>Screenshots</summary>
Add any relevant before/after screenshots here
</details>

## How did you test the change?

- [ ] iOS Simulator
- [ ] iOS Device
- [x] Android Simulator
- [ ] Android Device
- [ ] `curl` to a dev App Engine server
- [ ] other, please describe

## Checklist:

- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md) and verified that all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE).

### CODE REVIEW

File: client/flutter/assets/content_bundles/protect_yourself.en.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ contents:
         <b>Cover your mouth and nose</b> with your bent elbow or tissue when you cough or sneeze.
     - image_name: protect-distance
       body_html: |
-        <b>Stay more than</b> 1 meter (3 feet) away from a person who is sick.
+        <b>Stay more than 1 meter</b> (3 feet) away from other people when possible.
```
Comment (advayDev1) [inline client/flutter/assets/content_bundles/protect_yourself.en.yaml:17]: This is a change in medical content and should be reviewed by WHO @borowiecth

File: client/flutter/assets/content_bundles/protect_yourself.en.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ contents:
...
     - image_name: protect-mask
       body_html: |
-        Only wear a mask if you or someone you are looking after are ill with COVID-19 symptoms (especially coughing).
+        <b>Wear masks closer than 1 meter</b> (3 feet) in areas where the virus is spreading.
```
Comment (advayDev1) [inline client/flutter/assets/content_bundles/protect_yourself.en.yaml:20]: This is a change in medical content and should be reviewed by WHO @borowiecth .

The grammar doesn't seem to make sense to me.

File: client/flutter/assets/content_bundles/protect_yourself.en.yaml (hunk 1 above)
Comment (borowiecth) [reply client/flutter/assets/content_bundles/protect_yourself.en.yaml:17]: Maintain at least a 1-metre distance between yourself and others. Maintain an even greater distance when indoors. 
Can be shortened to:

> Maintain at least a 1-metre distance with others and an even greater distance when indoors. 

https://www.who.int/emergencies/diseases/novel-coronavirus-2019/question-and-answers-hub/q-a-detail/q-a-on-covid-19-and-masks

File: client/flutter/assets/content_bundles/protect_yourself.en.yaml (hunk 2 above)
Comment (borowiecth) [reply client/flutter/assets/content_bundles/protect_yourself.en.yaml:20]: I may be an acceptable rephrasing of:

> Within wider environments where the virus is spreading, masks should be worn by the general public in settings where it is not possible to maintain at least 1 meter from others.
But on the main page there is a less controversial statement:

> Maintain at least a 1-metre distance between yourself and others
I suggest this one instead

But we should remove the sentence "Only wear a mask if you or someone you are looking after...."
And replace it by this statement that is repeated regularly in the press meetings:

> Avoid the 3Cs: spaces that are Closed, Crowded or involve Close Contact.

File: client/flutter/assets/content_bundles/protect_yourself.en.yaml (hunk 1 above)
Comment (brunobowden) [reply client/flutter/assets/content_bundles/protect_yourself.en.yaml]: @borowiecth - I'd propose:

"**Distance at least a 1 meter** (3 feet) with others and an even greater distance when indoors"

This allows you to put emphasis on "Distance" and "1 meter" in a smaller bolded section whereas "Maintain" isn't so informative.

There's a separate question about "meter" vs. "metre". The WHO website uses both with slightly more usage of "metre". Around the internet in general, there's a 3X preference for "meter" when it comes to associations with COVID:
13M: "1 meter" covid - [Google Search]https://www.google.com/search?q=%221+meter%22+covid)
4M: "1 metre" covid - [Google Search]https://www.google.com/search?q=%221+metre%22+covid)

File: client/flutter/assets/content_bundles/protect_yourself.en.yaml (hunk 2 above)
Comment (brunobowden) [reply client/flutter/assets/content_bundles/protect_yourself.en.yaml:20]: @borowiecth - this section has an image of a mask, so talking about the 3 C's doesn't match up. Perhaps we should have another tile for advice like that. We should also have some specific guidance around masks as that's an important public health measure.

Comment (brunobowden) [reply client/flutter/assets/content_bundles/protect_yourself.en.yaml:20]: @advayDev1 - would this read better to you?

"<b>Wear masks when closer than 1 meter</b> (3 feet) in areas where the virus is spreading."

Comment (advayDev1) [reply client/flutter/assets/content_bundles/protect_yourself.en.yaml:20]: Yeah grammar LGTM with that if WHO approves the content itself.

Comment (borowiecth) [reply client/flutter/assets/content_bundles/protect_yourself.en.yaml:20]: OK for this

**Comment by brunobowden** [comment]
@theswerd @yukuairoy - please can one of you approve, just a couple of lines

File: client/flutter/assets/content_bundles/protect_yourself.en.yaml [hunk 3]
```diff
@@ -14,7 +14,7 @@ contents:
         <b>Cover your mouth and nose</b> with your bent elbow or tissue when you cough or sneeze.
     - image_name: protect-distance
       body_html: |
-        <b>Stay more than</b> 1 meter (3 feet) away from a person who is sick.
+        <b>Distance at least a 1 meter</b> (3 feet) with others and an even greater distance when indoors.
```
Comment (yukuairoy) [inline client/flutter/assets/content_bundles/protect_yourself.en.yaml:17]: Can you remove the extra "a" in "a 1 meter"

**Review by yukuairoy — CHANGES_REQUESTED** [review]

File: client/flutter/assets/content_bundles/protect_yourself.en.yaml (hunk 3 above)
Comment (brunobowden) [reply client/flutter/assets/content_bundles/protect_yourself.en.yaml]: Good catch. That shouldn't be there.

**Review by yukuairoy — APPROVED** [review]

**Pull request merged**